SU2_CFD/include/solvers/CFVMFlowSolverBase.hpp

```diff
@@ -318,13 +339,7 @@ class CFVMFlowSolverBase : public CSolver {
      *    Critical sections are used for this instead of reduction
      *    clauses for compatibility with OpenMP 2.0 (Windows...). ---*/
 
-    SU2_OMP_MASTER
-    {
-      Min_Delta_Time = 1e30;
-      Max_Delta_Time = 0.0;
-      Global_Delta_UnstTimeND = 1e30;
-    }
-    SU2_OMP_BARRIER
+    ompMasterAssignBarrier(Min_Delta_Time,1e30, Max_Delta_Time,0.0, Global_Delta_UnstTimeND,1e30);
```

Comment:
Not sure if I made it better or worse :+1: :-1:

Reply:
saved some lines :+1: nice C++ features :+1: fewer end macros to maintain :+1:
made the code hard to read without looking up the definition of `ompMasterAssignBarrier` :-1: